src/Controls/tests/DeviceTests/Elements/Modal/ModalTests.cs

```diff
@@ -556,6 +556,28 @@ await CreateHandlerAndAddToWindow(window, async () =>
 			});
 		}
 
+#if WINDOWS || MACCATALYST
```

Comment:
Why is not running in the rest of the platforms?

Reply:
@jsuarezruiz This issue only occurs when closing the window, which is a scenario supported on Windows and MacCatalyst. On Android and iOS, the concept of closing a window does not exist in the same way, so this behavior cannot be reproduced or tested on those platforms. For this reason, the test is limited to Windows and MacCatalyst.